cvat-canvas/src/canvas.ts

```diff
@@ -1,5 +1,92 @@
+/*
+* Copyright (C) 2018 Intel Corporation
+* SPDX-License-Identifier: MIT
+*/
 
-function tmp(message: string) {
-    console.log(message)
-    return message;
+/* eslint-disable */
+// Temporary disable eslint
+
+interface CanvasInterface {
+    html(): HTMLElement;
+    setup(frameData: any, objectStates: any[]): void;
+    activate(clientID: number, attributeID?: number): void;
+    rotate(direction: Rotation): void;
+    focus(clientID: number, padding?: number): void;
+    fit(): void;
+    grid(stepX: number, stepY: number, color?: GridColor, opacity?: number): void;
+
+    draw(shapeType: string, numberOfPoints: number, initialState: any): any;
+    split(enabled?: boolean): any;
+    group(enabled?: boolean): any;
+    merge(enabled?: boolean): any;
+
+    cancel(): void;
+}
+
+export enum GridColor {
+    BLACK,
```

Comment:
Again, I prefer to have CSS for grid instead of an explicit argument. Maybe I'm wrong but it is better to be consistent.

Reply:
Done